ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-models.j2

```diff
@@ -0,0 +1,16 @@
+{# Management API Models #}
+{% if management_api_models is arista.avd.defined %}
+!
+management api models
+{%     for provider in management_api_models.providers | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```

Comment:
```suggestion
{%     for provider in management_api_models.providers | arista.avd.natural_sort('name') %}
```

Reply:
We don't need ```arista.avd.convert_dicts('name')``` since you are building a new model with a list of dictionaries.

Reply:
👍 thanks Tony was not clear `arista.avd.convert_dicts` being only used for refactoring. Will update template as suggested.